themes/default/content/templates/static-website/aws/index.md

```diff
@@ -0,0 +1,802 @@
+---
+title: AWS Static Website
+meta_desc: The AWS Static Website template makes it easy to deploy a static website on AWS with Pulumi, Amazon S3, and Amazon CloudFront.
+meta_image: meta.png
+card_desc: Deploy a static website on AWS with Pulumi, Amazon S3, and Amazon CloudFront.
+layout: template
+template:
+    prefix: static-website-aws
+cloud:
+  name: Amazon Web Services
+  slug: aws
+---
+
+The AWS Static Website template deploys an HTML-based website on AWS with Pulumi. It uses an [Amazon S3 bucket]({{< relref "/registry/packages/aws/api-docs/s3/bucket" >}}) for file storage, configures the bucket to host a website, and deploys an [Amazon CloudFront Distribution]({{< relref "/registry/packages/aws/api-docs/cloudfront/distribution" >}}) to serve the website with low latency, caching, and HTTPS. The template generates a complete Pulumi program, including placeholder web content, to give you working project out of the box that you can customize easily and extend to suit your needs.
+
+![An architecture diagram of the Pulumi AWS Static Website template](./architecture.png)
+
+## Using this template
+
+To use this template to deploy a website of your own, make sure you've [installed Pulumi]({{< relref "/docs/get-started/install" >}}) and [configured your AWS credentials]({{< relref "/registry/packages/aws/installation-configuration" >}}), then create a new [project]({{< relref "/docs/intro/concepts/project" >}}) using the template in your language of choice:
+
+{{% chooser language "typescript,python,go,csharp,yaml" / %}}
+
+{{% choosable language typescript %}}
+
+```bash
+$ mkdir my-site && cd my-site
+$ pulumi new static-website-aws-typescript
+```
+
+{{% /choosable %}}
+
+{{% choosable language python %}}
+
+```bash
+$ mkdir my-site && cd my-site
+$ pulumi new static-website-aws-python
+```
+
+{{% /choosable %}}
+
+{{% choosable language go %}}
+
+```bash
+$ mkdir my-site && cd my-site
+$ pulumi new static-website-aws-go
+```
+
+{{% /choosable %}}
+
+{{% choosable language csharp %}}
+
+```bash
+$ mkdir my-site && cd my-site
+$ pulumi new static-website-aws-csharp
+```
+
+{{% /choosable %}}
+
+{{% choosable language yaml %}}
+
+```bash
+$ mkdir my-site && cd my-site
+$ pulumi new static-website-aws-yaml
+```
+
+{{% /choosable %}}
+
+Follow the prompts to complete the new-project wizard. When it's done, you'll have a complete Pulumi project configured with the most common settings, ready to deploy. Feel free to inspect the code in {{< langfile >}} for a closer look.
```

Comment:
should "new-project" have the hyphenation?

Reply:
Yep, as it's an adjectival phrase.